### PR TITLE
json-glib 1.4.2

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

File: Formula/json-glib.rb [hunk 1]
```diff
@@ -13,15 +12,20 @@ class JsonGlib < Formula
 
   depends_on "gobject-introspection" => :build
   depends_on "pkg-config" => :build
+  depends_on "meson" => :build
```
Comment (ilovezfs) [inline Formula/json-glib.rb]: not meson-internal?

Comment (tschoonj) [reply Formula/json-glib.rb]: Thanks!

**Comment by fxcoudert** [comment]
Why do we need to patch, and is upstream working on fix it?

**Comment by tschoonj** [comment]
The lines i remove in atk and Json-glib are actually hacks that were put in to add versioning support when building dylibs, an essential feature that is missing in meson. The recently added meson-internal formula, used in atk and json-glib, contains a patch that adds proper dylib versioning, which is why I removed these hacks.

**Comment by tschoonj** [comment]
Oh and the version numbers in some of those hacks are wrong anyway in my experience

**Comment by ilovezfs** [comment]
🍨

**Pull request closed without merging**